### DISCUSSION TITLE
bg-clip-text do not work and fix it this way

### DISCUSSION THREAD

**Discussion by asionesjia** [tailwindlabs/tailwindcss#15387] [Help] [answered]
Fix your this problem simply add: `-webkit-background-clip: text;`

**Comment by philipp-spiess** [comment]
Hey! What browser are you seeing issues with? According to https://caniuse.com/background-clip-text it's supported without a prefix from Chrome 120+

**Reply by philipp-spiess** [reply, marked as answer]
Just a heads-up that we're changing the Chrome target so that `-webkit-background-clip` is now added, see: https://github.com/tailwindlabs/tailwindcss/pull/15389

This change went out with the latest beta update just now :)
